Lib/glyphsLib/builder/axes.py

```diff
@@ -322,6 +324,51 @@ def font_uses_axis_locations(font):
     )
 
 
+def _potentially_invert_mapping(self, mapping, instance_mapping):
+    # Fontmake introduced an Axis Mapping custom parameter before Glyphs did;
+    # then Glyphs introduced it, but with an inverted interpretation. Then
+    # Glyphs changed the interpretation to match fontmake's. Try and figure out
```

Comment:
when we say "Glyphs changed the interpretation to match fontmake's", does that mean it simply took the existing parameter's keys are meaning user/external and the values design/internal but keep them as were originally in the source... or does it mean that it suddenly decided to swap them?
Is this PR attempting to un-flip them?!

Reply:
It literally swaps them.